xml/System.Web.DynamicData/DynamicFilter.xml

```diff
@@ -417,9 +417,9 @@
         <Parameter Name="source" Type="System.Linq.IQueryable" Index="0" FrameworkAlternate="netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
       </Parameters>
       <Docs>
-        <param name="source">The data source query.</param>
-        <summary>Returns the filter repeater.</summary>
-        <returns>The filter.</returns>
+        <param name="source">To be added.</param>
+        <summary>To be added.</summary>
+        <returns>To be added.</returns>
```

Comment:
@gewarren Changes like this do concern me. Any ideas why this happens?

Reply:
Good catch, this is a bug. I've logged https://dev.azure.com/ceapex/Engineering/_workitems/edit/598286.